packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -311,19 +311,15 @@
       },
     },
     $trs: {
-      toolbarHeader: {
-        message: 'Edit device sync schedule',
-        context: 'Heading for edit schedule page.',
-      },
-      editSyncScheduleSubTitle: {
+      editSyncScheduleTitle: {
         message: 'Edit device sync schedule',
         context: 'Subtitle for the edit sync schedule page',
       },
       serverTime: {
         message: 'Server time:',
         context: 'Server time label',
       },
-      checboxlabel: {
+      checkboxlabel: {
         message: 'If scheduled sync fails, retry until the next scheduled sync',
```

Comment:
Didn't we decide for this to be: `If scheduled sync fails, keep trying`?

Reply:
To go one further `checkboxLabel` to use camelCase?